src/EditorFeatures/Core/Implementation/NavigateTo/NavigateToSymbolFinder.cs

```diff
@@ -19,7 +19,6 @@ internal static class NavigateToSymbolFinder
         internal static async Task<IEnumerable<ValueTuple<DeclaredSymbolInfo, Document, IEnumerable<PatternMatch>>>> FindNavigableDeclaredSymbolInfos(Project project, string pattern, CancellationToken cancellationToken)
```

Comment:
Can "DotArray" be deleted now?
(This comment should be on line 17, but I can't figure out how to put it there since it wasn't close enough to any line modified by this change.)